docs/trustee-tpm-setup.md

```diff
@@ -0,0 +1,203 @@
+# Introduction
+
+Instructions to use TPM attester for remote attestation with Trustee. This is primarily meant to
+experiment with Trustee attestation and resource policies based on TPM PCRs
+which can then be adapted for a confidential environment (eg. Azure CVM with vTPM)
+
+## Trustee server setup
+
+### Setup services
+
+Ensure you have `docker`, `docker-compose`, `rust`, `make` installed.
+
+Clone the repo.
+
+```sh
+git clone https://github.com/bpradipt/trustee.git
+cd trustee
+git checkout -b tpm-verifier origin/tpm-verifier
+```
+
+Setup authentication keys.
+We'll need these keys when using `kbs-client` to interact with Trustee services.
+
+```sh
+openssl genpkey -algorithm ed25519 > kbs/config/private.key
+openssl pkey -in kbs/config/private.key -pubout -out kbs/config/public.pub
+```
+
+Run Trustee services.
+
+This will start KBS, AS and RVPS services
+
+```sh
+docker-compose up -d
+```
+
+### Build and install kbs-client
+
+```sh
+cd kbs
+make cli
+sudo make install-cli
+```
```

Comment:
Can you please mention how to create the VM, or at least the README where to find the instructions

Reply:
Yes, just refer to the [README](https://github.com/confidential-clusters/investigations?tab=readme-ov-file#start-fcos-vm)